content/donate/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Donate
+description: "Donate to Techqueria and support our efforts to elevate Latinx in Tech."
+url: /donate/
+aliases:
+  - /donations/
+  - /give-money/
+categories:
+  - sitemap
+---
+
+Techqueria is a nonprofit representing one of the largest communities for Latinx professionals in the tech industry.
+
+Techqueria is a 501c3 nonprofit and all donations are tax deductible.
```

Comment:
Use this copy whenever we mention tax deductions: 
"Techqueria (EIN 83-2656715) is a tax-exempt public charity under Section 501(c)(3) of the Internal Revenue Code. Contributions to Techqueria are tax deductible to the extent provided by law."